### PR TITLE
BusinessDashboard Theme | <Popover> - Changed colors

### PR DESCRIPTION
**Pull request by shelly-goldblit**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6991/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6991

**Review by adicicurel — CHANGES_REQUESTED** [review]
Add visual tests

File: packages/wix-style-react/src/Popover/test/Popover.visual.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import React, { useState, useEffect } from 'react';
+import { visualize } from 'storybook-snapper';
+
```
Comment (adicicurel) [inline packages/wix-style-react/src/Popover/test/Popover.visual.js:3]: remove empty line

**Review by wix-fed-exam — APPROVED** [review]

**Review by adicicurel — APPROVED** [review]

File: packages/wix-style-react/src/Themes/businessDashboard/components/Popover.st.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  -st-from: "../../../Foundation/stylable/shadows.st.css";
+  -st-named: shadow30;
+}
+
+:import {
+  -st-from: "../foundations/colors.st.css";
+  -st-named: D20;
+}
+
+:import {
```
Comment (mykas) [inline packages/wix-style-react/src/Themes/businessDashboard/components/Popover.st.css:16]: There is no need for this import. The arrow and popoverContent are already available in Popover stylesheet which you can override. Just like you did here: https://github.com/wix/wix-style-react/blob/master/packages/wix-style-react/src/Page/PageSticky.js#L35

**Pull request merged**